include/tlog/source.h

```diff
@@ -32,7 +32,8 @@
 
 /** Abstract source */
 struct tlog_source {
-    const struct tlog_source_type *type;   /**< Type */
+    const struct tlog_source_type *type;          /**< Type */
+    struct timespec               last_timestamp; /**< Timestamp */
```

Comment:
Could you please align the parameter names? I.e. add a space before `last_timestamp`?

Reply:
Do you mean a space between timespec and last_timestamp? I think *type and last_timestamp are same type ( variable) that is reason why I align these parameter names